contracts/governance/AdminMultisigBase.sol

```diff
@@ -0,0 +1,219 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.8.9;
+
+import { IAdminMultisig } from '../interfaces/IAdminMultisig.sol';
+import { EternalStorage } from '../EternalStorage.sol';
+
+contract AdminMultisigBase is EternalStorage, IAdminMultisig {
```

Comment:
Need to think of a different name, since these addresses are more like co-signers rather than governance admins